packages/device-management-kit/src/api/device-action/os/OpenAppDeviceAction/OpenAppDeviceAction.ts

```diff
@@ -126,7 +126,12 @@ export class OpenAppDeviceAction extends XStateDeviceAction<
           if (context._internalState.currentlyRunningApp === null)
             throw new Error("context.currentlyRunningApp === null");
           return (
-            context._internalState.currentlyRunningApp === context.input.appName
+            context._internalState.currentlyRunningApp ===
+              context.input.appName ||
+            (context.input.compatibleAppNames !== undefined &&
+              context.input.compatibleAppNames.includes(
+                context._internalState.currentlyRunningApp,
+              ))
```

Comment:
[COULD] be more concise
```suggestion
            (context.input.compatibleAppNames?.includes(
                context._internalState.currentlyRunningApp,
              ))
```

Reply:
Got a TS error using this syntax: `Type boolean | undefined is not assignable to type boolean.`

Reply:
Casting the type to a boolean may be able to remove the warning, but I think the original one is OK as well.